modules/dnn/test/test_layers_1d.cpp

```diff
@@ -245,4 +245,37 @@ INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Elemwise_1d_Test, Combine(
 /*operation*/           Values("div", "prod", "max", "min", "sum")
 ));
 
+typedef testing::TestWithParam<tuple<std::vector<int>>> Layer_Const_Test;
+TEST_P(Layer_Const_Test, Accuracy)
+{
+    std::vector<int> input_shape = get<0>(GetParam());
+
+    LayerParams lp;
+    lp.type = "Const";
+    lp.name = "ConstLayer";
+
+    Mat constBlob, output_ref;
+    if (input_shape.size() > 1)
+        constBlob = Mat(input_shape, CV_32F);
+    else
+        constBlob = Mat(input_shape[0], input_shape.data(), CV_32F);
```

Comment:
```suggestion
constBlob = Mat(input_shape.size(), input_shape.data(), CV_32F);
```

Reply:
I have tried it, it does too work for `0D` case. Mat created like that can not filled with random value. I know it look ugly, but lets keep like it is

Reply:
Got it, thanks!